firefox-ios/Client/Frontend/TrackingProtection/BlockedTrackersTableController.swift

```diff
@@ -0,0 +1,385 @@
+                equalTo: backButton.trailingAnchor
+            ),
+            siteTitleLabel.trailingAnchor.constraint(
+                equalTo: closeButton.leadingAnchor
+            ),
+            siteTitleLabel.topAnchor.constraint(
+                equalTo: navigationView.topAnchor,
+                constant: TPMenuUX.UX.TrackingDetails.baseDistance
+            ),
+            siteTitleLabel.bottomAnchor.constraint(
+                equalTo: navigationView.bottomAnchor,
+                constant: -TPMenuUX.UX.TrackingDetails.baseDistance
+            ),
+
+            closeButton.trailingAnchor.constraint(
+                equalTo: navigationView.trailingAnchor,
+                constant: -TPMenuUX.UX.horizontalMargin
+            ),
+            closeButton.topAnchor.constraint(equalTo: navigationView.topAnchor,
+                                             constant: TPMenuUX.UX.horizontalMargin),
+            closeButton.heightAnchor.constraint(greaterThanOrEqualToConstant: TPMenuUX.UX.closeButtonSize),
+            closeButton.widthAnchor.constraint(greaterThanOrEqualToConstant: TPMenuUX.UX.closeButtonSize),
+            closeButton.widthAnchor.constraint(equalTo: closeButton.heightAnchor),
+
+            horizontalLine.leadingAnchor.constraint(equalTo: navigationView.leadingAnchor),
+            horizontalLine.trailingAnchor.constraint(equalTo: navigationView.trailingAnchor),
+            horizontalLine.bottomAnchor.constraint(equalTo: navigationView.bottomAnchor),
+            horizontalLine.heightAnchor.constraint(equalToConstant: TPMenuUX.UX.Line.height),
+        ]
+
+        constraints.append(contentsOf: navigationViewContraints)
+    }
+
+    // MARK: TableView Setup
+    private func setupTableView() {
+        view.addSubview(blockedTrackersTableView)
+        let tableConstraints = [
+            blockedTrackersTableView.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+                constant: TPMenuUX.UX.horizontalMargin
+            ),
+            blockedTrackersTableView.topAnchor.constraint(
+                equalTo: navigationView.bottomAnchor,
+                constant: TPMenuUX.UX.BlockedTrackers.headerDistance
+            ),
+            blockedTrackersTableView.bottomAnchor.constraint(
+                greaterThanOrEqualTo: view.safeAreaLayoutGuide.bottomAnchor,
+                constant: 0
+            ),
+            blockedTrackersTableView.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                constant: -TPMenuUX.UX.horizontalMargin
+            )
+        ]
+        constraints.append(contentsOf: tableConstraints)
+    }
+
+    private func setupDataSource() {
+        // swiftlint:disable line_length
+        dataSource = UITableViewDiffableDataSource<Int, BlockedTrackerItem>(tableView: blockedTrackersTableView) { (tableView, indexPath, item) -> UITableViewCell? in
+        // swiftlint:enable line_length
```

Comment:
why are we disabling line length here instead of moving this into multiple lines, not sure when we should disable so tagging roux to also verify. I see we are also disabling it here `SearchGroupedItemsViewController` but don't have context on why. 

cc: @adudenamedruby

Reply:
I would personally multi-line the heck outta this, yeah.